packages/knip/src/plugins/expo/helpers.ts

```diff
@@ -0,0 +1,81 @@
+import type { PluginOptions } from '../../types/config.js';
+import { type Input, toDependency, toProductionDependency } from '../../util/input.js';
+import { getPackageNameFromModuleSpecifier } from '../../util/modules.js';
+import type { ExpoConfig } from './types.js';
+
+// https://docs.expo.dev/versions/latest/config/app
+
+export const getDependencies = async (expoConfig: ExpoConfig, { manifest }: PluginOptions) => {
+  const config = 'expo' in expoConfig ? expoConfig.expo : expoConfig;
+
+  const platforms = config.platforms ?? ['ios', 'android'];
+
+  const pluginPackages =
+    (config.plugins
+      ?.map(plugin => {
+        const pluginName = Array.isArray(plugin) ? plugin[0] : plugin;
+        return getPackageNameFromModuleSpecifier(pluginName);
+      })
+      .filter(Boolean) as string[]) ?? [];
+
+  const inputs = new Set<Input>([
+    ...['expo', 'react-native', 'react'].map(toProductionDependency),
```

Comment:
> Adds `expo`, `react`, and `react-native` as dependencies; required for all Expo projects.

Aren't those dependencies referenced in regular ways already? I'd think it isn't necessary to add them explicitly. It's a bit far-fetched, but in theory this might change, and if that's what they are, peer dependencies should picked up properly already.

Reply:
Ahh I didn't realize Knip picked up peer dependencies. Very cool! That takes care of `react` and `react-native`, and on second thought `expo` will almost certainly be in the package.json scripts (if not imported in the project), so I'll remove that as well.